src/style.css

```diff
@@ -1 +1,133 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+:root {
+  --font-family: Avenir, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic font family as a fallback. For example, you could modify the variable to `--font-family: Avenir, Arial, sans-serif;`.